### DISCUSSION TITLE
pyvista.Text3D How to set font file

### DISCUSSION THREAD

**Discussion by zengzhengrong** [pyvista/pyvista#4731] [Q&A]
I need to use other language display

**Comment by MatthewFlamm** [comment]
This might be in progress in #4639, but pinging @tkoyama010 in case this request is different than the PR.

**Reply by tkoyama010** [reply]
We are adding the ability to add fonts using `vtkTextActor`. So it is different to support `Text3D` using `vtkVectorText` . To achieve this, it is necessary to investigate how `vtkVectorText` manages fonts.
In any case, thank you for your request.
